readme.txt

```diff
@@ -34,7 +33,25 @@ Answer to foo bar dilemma.
 
 == Changelog ==
 
+= 0.6 =
+* Admin Settings: Added notices for when settings are saved or reset
+* Site Health: Added Site Health option

```

Comment:
Oops, this one is actually part of the issue template changes and can be removed from the changelog. My mistake.